test/fuzz/actors_setup_test.go

```diff
@@ -509,23 +616,38 @@ func simulateAutomaticInitialState(
 	data *SimulationData,
 ) (iterationCountAfter int) {
 	iterationCount := 0
-	listTopics := []uint64{1, 2}
 
 	// make sure that the setup always fails on error
 	failOnErrWanted := data.failOnErr
 	data.failOnErr = true
 
 	// additive actions
 
-	// create two topics
-	success := createTopic(m, faucet, UnusedActor, nil, 0, data, iterationCount)
+	// pick 4 reputers, 5 workers, and 2 delegators
+	startReputers, startWorkers, startDelegators := pickAutoSetupActors(m, data)
+
+	// add 2 reputers in admin whitelist
+	iterationCount = startAddToAdminWhitelist(m, data, faucet, startReputers[2:], iterationCount)
+	// put 2 reputers & 2 workers in global whitelist
+	iterationCount = startAddToGlobalWhitelist(m, data, faucet, append(startReputers[:2], startWorkers[:2]...), iterationCount)
+	// add 2 delegators to create topic whitelist
+	iterationCount = startAddToTopicCreatorWhitelist(m, data, faucet, startDelegators, iterationCount)
+
+	// create two topics with both reputers and workers whitelist enabled
+	success := createTopic(m, startDelegators[0], UnusedActor, nil, 0, data, iterationCount)
 	require.True(m.T, success)
 	iterationCount++
-	success = createTopic(m, faucet, UnusedActor, nil, 0, data, iterationCount)
+	success = createTopic(m, startDelegators[1], UnusedActor, nil, 0, data, iterationCount)
 	require.True(m.T, success)
 	iterationCount++
-	// pick 4 reputers, 4 workers, and 2 delegators
-	startReputers, startWorkers, startDelegators := pickAutoSetupActors(m, data)
+
+	listTopics := data.getTopics()
+	require.Len(m.T, listTopics, 2)
+
+	// put 2 reputers & 2 workers in one topic whitelists
+	iterationCount = startAddToTopicWorkerWhitelist(m, data, faucet, startWorkers[2:], listTopics[1], iterationCount)
+	iterationCount = startAddToTopicReputerWhitelist(m, data, faucet, startReputers[2:], listTopics[1], iterationCount)
```

Comment:
Are we running through every possible state transition at least once? If not can you change this code so that it runs through every state transition that you have added, including the `removal` state transitions?

so e.g. instead of putting 2 reputers and 2 workers, you would put 3 in and remove 1 before the simulation fully started

Reply:
Oh ok I'll make sure we run through every transition at least once, this is clearly not the case right now